src/components/StoryList.component.js

```diff
@@ -38,8 +38,14 @@ class StoryList extends Component {
   }
 
   onClickDelete(id) {
-    //<--TODO: Add delete message
-    console.log('Delete Button #' + id)
+    const { selectedEvent, loadStories } = this.props
+    if (
+      window.confirm(
+        'Delete the current story? This will permanently remove the story from the story-list.'
+      )
+    ) {
+      Api.deleteStory(id).then(() => loadStories())
```

Comment:
If we delete a story that is currently selected, the Exit Story button is still displayed. Since it doesn't make sense to have a story selected that no longer exists, we should also exit the story if it's the one that's currently selected. Check `exitStory()` in `App.js`